js/parts-more/RadialAxis.js

```diff
@@ -436,15 +461,21 @@ addEvent(Axis, 'init', function (e) {
     }
     else if (polar) {
         extend(this, radialAxisMixin);
-        isCircular = isX;
-        this.defaultRadialOptions = isX ?
-            this.defaultRadialXOptions :
-            merge(this.defaultYAxisOptions, this.defaultRadialYOptions);
+        // Check which axis is circular
+        isCircular = !inverted && coll === 'xAxis' ||
+            inverted && coll === 'yAxis';
+        this.defaultRadialOptions = isCircular ?
+            this.defaultCircularRadialOptions :
+            merge(this.defaultYAxisOptions, this.defaultNonCircularRadialOptions);
+        // Prevents from rendering an additional undesirable grid line
+        // and label
+        if (!isCircular) {
+            this.defaultRadialOptions.minPadding = 0;
+        }
```

Comment:
Why the IF is not in merge?

Reply:
I can see now that this should apply to xAxis on inverted polar chart only to prevent from rendering undesirable grid line and label (-1 in case of default options). Made required changes.

Reply:
On a category X axis the `minPadding` should be 0, maybe the problem is somewhere else?

Reply:
The additional tick (-1) was a result of merging the `defaultNonCircularRadialOptions` object with the `defaultYAxisOptions` where minPadding is set to 0.05 and `startOnTick` is set to true no matter if vertical axis was X or Y type. Now the `defaultNonCircularRadialOptions` is merged into a correct set of options (the `defaultOptions` in case of `xAxis` and the `defaultYAxisOptions` in case of `yAxis`).